src/api/grpc/transport.grpc.api.ts

```diff
@@ -49,6 +49,19 @@ const authInterceptor: Interceptor =
 				logoutFunction(false);
 			}
 
+			if (error.code === Code.ResourceExhausted) {
```

Comment:
From local test, I see that when we get rate-limit, it returns `error.code=14` (Unavailable).
I'll check with Efi and will give you an answer.